### DISCUSSION TITLE
I can't verify I am a student

### DISCUSSION THREAD

**Discussion by ghost** [community/community#51383] [GitHub Education]
Hi, I am a student. I upload the documents that I can get from my university. But your system doesn't verify. I have lots of membership with my student account. You are the first one who doesn't verify automatically. You can check my status with qr-code.
Moderator's EDIT: I've removed your private information from [this post](https://github.com/orgs/community/discussions/51383) as posting identifiable information is not allowed, per our [Code of Conduct](https://github.com/community/community/blob/main/CODE_OF_CONDUCT.md).

**Comment by F0XN0T** [comment]
I understand that you're experiencing challenges in accessing the GitHub Student Developer Pack, and it's concerning to hear that you haven't received a satisfactory response from GitHub. It's important for platforms to handle personal information with care and respect user privacy while also providing prompt and compassionate support.

If you're still encountering difficulties, you could consider reaching out to GitHub's customer support team and providing a detailed explanation of the situation. You might also find it beneficial to connect with other students and creators who may be experiencing similar issues to explore potential collective solutions.

I hope that a resolution is found promptly as it's affecting many users and requires urgent attention.

**Comment by ghost** [comment]
I solved it with new document from my school. Thanks.
